packages/react-native-reanimated/Common/cpp/reanimated/Tools/ReanimatedProfilingSection.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+//#define REANIMATED_PROFILING
+
+#ifdef REANIMATED_PROFILING
+
+#ifdef ANDROID
+#include <android/trace.h>
+#else
+#include <os/signpost.h>
+const os_log_t log_handle = os_log_create("reanimated", OS_LOG_CATEGORY_POINTS_OF_INTEREST);
+#endif
+
+#endif
+
+namespace reanimated {
+
+// Convenience macros that allow for easy marking of relevant sections both on Android and iOS
+
+#ifdef REANIMATED_PROFILING
+
+#ifdef ANDROID
+
+// On android we use ATrace. The easiest way to produce a trace is to run profiling through Android Studio with System Trace recording.
+// The result can be viewed through Perfetto web UI.
+
+#define PROFILER_START(x) ATrace_beginSection(#x);
+
+#define PROFILER_END(x) ATrace_endSection();
+
+#else
```

Comment:
Would it be correct?
```suggestion
#else // iOS & MacOS
```